### PR TITLE
Update Jekyll to 3.7.4

### PR DESCRIPTION
**Pull request by naoisegolden**
Fixes issue where dependency concurrent-ruby-1.1.0 was no found.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ source "https://rubygems.org"
 #
 # This will help ensure the proper Jekyll version is running.
 # Happy Jekylling!
-gem "jekyll", "~> 3.7.3"
+gem "jekyll", "~> 3.8.5"
```
Comment (enricostano) [inline Gemfile]: Is this version supported by GitHub Pages?

Comment (sauloperez) [reply Gemfile]: We no longer rely on GH's pages build. We handle it ourselves.

Comment (sauloperez) [reply Gemfile]: Looks like it supports up until 3.7.4. See https://pages.github.com/versions/

Comment (naoisegolden) [reply Gemfile]: Bummer

Comment (enricostano) [reply Gemfile]: So? Supported or not?

**Pull request merged**